modules/heat_conduction/src/materials/GapConductance.C

```diff
@@ -276,12 +279,31 @@ GapConductance::computeQpConductance()
   }
 }
 
+Real
+GapConductance::gapAttenuation(Real adjusted_length, Real min_gap, unsigned int min_gap_order)
+{
+  if (adjusted_length > min_gap)
+    return 1.0 / adjusted_length;
```

Comment:
`mooseAssert` on adjusted_length?

Reply:
Added an assert on `min_gap` (that protects the `adjusted_length` branch as well due to the `adjusted_length > min_gap` condition). I also changed the range checking to ensure that `min_gap` is always strictly larger than zero.

Reply:
I don't know why this didn't get marked as outdated, the change is there.